### PR TITLE
Label Improvment: Change size and default displayed Info

### PR DESCRIPTION
**Pull request by RomanenkoVladimir**
# Label Improvment: Change size and default displayed Info

_**Please read the [CONTRIBUTING.md](https://github.com/MaibornWolff/codecharta/blob/main/CONTRIBUTING.md) before opening a PR.**_

## Description

_(Currently there are a lot of label change requests, I will split my improvment suggestions into smaller PRs to make reviews easier)_

This PR will:
- Remove metrics from being displayed by default
- Fix width issues with mutli-line labels, making them a bit more compact

### CODE REVIEW

File: visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts [hunk 1]
```diff
@@ -118,7 +118,10 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 		const multiLineContext = message.split("\n")
 
 		// setting canvas width/height before ctx draw, else canvas is empty
-		canvas.width = context.measureText(message).width + margin
+		canvas.width =
+			context.measureText(multiLineContext[0]).width > context.measureText(multiLineContext[1]).width
+				? context.measureText(multiLineContext[0]).width + margin
+				: context.measureText(multiLineContext[1]).width + margin
```
Comment (BridgeAR) [inline visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts:124]:
```suggestion
		const firstMultiLineContextWidth = context.measureText(multiLineContext[0]).width
		const secondMultiLineContextWidth = context.measureText(multiLineContext[1]).width
		canvas.width =
			firstMultiLineContextWidth > secondMultiLineContextWidth
				? firstMultiLineContextWidth + margin
				: secondMultiLineContextWidth + margin
```

Comment (RomanenkoVladimir) [reply visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts:124]: That is more performant, thanks for the feedback

**Review by BridgeAR — COMMENTED** [review]
LGTM. Please add a changelog entry though :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -39,16 +41,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
...
     -   Numbers are sorted naturally
     -   Characters are compared with their base character (e.g., `a` is now next to `á`).
+-   Label metric not shown by default
 
 ### Removed 🗑
 
 ### Fixed 🐞
 
+-   Label showing node name and metric values (2 lines) have incorrect width
+
```
Comment (BridgeAR) [inline CHANGELOG.md:57]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -39,16 +41,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
     1. Whitespace handling changed to ignore leading whitespace.
 
 -   Distribution metric #1188
+
     1. set rloc to default distribution metric, showing language percentages for real lines of code, if available. Else set to unary to show language distribution over files
 
 -   Improved file sorting in the file overview of the search bar
     -   Numbers are sorted naturally
     -   Characters are compared with their base character (e.g., `a` is now next to `á`).
+-   Label metric not shown by default
```
Comment (BridgeAR) [inline CHANGELOG.md:50]:
```suggestion
-   Label metric not shown by default anymore
```

**Review by BridgeAR — APPROVED** [review]

**Pull request merged**